rsciio/quantumdetector/_api.py

```diff
@@ -607,7 +581,6 @@ def file_reader(
         navigation_shape=navigation_shape,
         first_frame=first_frame,
         last_frame=last_frame,
-        distributed=distributed,
```

Comment:
I think you might need to have a **kwargs argument so that having distributed=True doesn't cause an error.  We probably don't want to break a bunch of peoples code?

Reply:
Done!